package.json

```diff
@@ -32,6 +36,7 @@
     "tslib": "^2.3.1"
   },
   "devDependencies": {
+    "@abhijeetnarvekar/ckeditor5-build-super": "^35.0.2",
```

Comment:
@loaiabdelhalim Yes this can be certainly moved to @acrolinx. I don't have access to NPM account. I will create a separate ticket for it.

Does that work?

Reply:
Yes 👍🏻